### PR TITLE
xen-tools: Update package versions

### PR DESCRIPTION
**Pull request by temerkhanov**
Pull packages from Alpine 3.12 (too bad liburing only exists in edge)

Signed-off-by: Sergey Temerkhanov <s.temerkhanov@gmail.com>

### CODE REVIEW

**Review by eriknordmark — COMMENTED** [review]
Sergey, I don't have a problem moving to a new version but I wonder whether this will enable something new (or fix some bug).

**Comment by temerkhanov** [comment]
Erik, 3.8 is out of support since May https://wiki.alpinelinux.org/wiki/Alpine_Linux:Releases so we need to switch to a newer version anyway

File: pkg/xen-tools/Dockerfile.in [hunk 1]
```diff
@@ -67,17 +67,17 @@ WORKDIR /out/usr/lib/xen/bin/
...
+    bash=5.0.17-r0     \
+    libaio=0.3.112-r1  \
+    libbz2=1.0.8-r1    \
+    glib=2.64.5-r0     \
+    pixman=0.40.0-r2   \
+    yajl=2.1.0-r1      \
+    keyutils=1.6.1-r1  \
+    libusb=1.0.23-r0   \
+    xz-libs=5.2.5-r0
 RUN if [ "$(uname -m)" = "aarch64" ]; then apk add --no-cache libfdt=1.4.4-r0 ;fi
```
Comment (giggsoff) [inline pkg/xen-tools/Dockerfile.in:81]: For arm64 I receive:
```
Step 38/54 : RUN if [ "$(uname -m)" = "aarch64" ]; then apk add --no-cache libfdt=1.4.4-r0 ;fi
 ---> Running in 7944666c6d9e
fetch http://dl-cdn.alpinelinux.org/alpine/v3.12/main/aarch64/APKINDEX.tar.gz
fetch http://dl-cdn.alpinelinux.org/alpine/v3.12/community/aarch64/APKINDEX.tar.gz
ERROR: unsatisfiable constraints:
  libfdt-1.6.0-r0:
    breaks: world[libfdt=1.4.4-r0]
```

**Review by giggsoff — CHANGES_REQUESTED** [review]

File: pkg/xen-tools/Dockerfile.in (hunk 1 above)
Comment (temerkhanov) [reply pkg/xen-tools/Dockerfile.in]: Done

**Comment by rvs** [comment]
Hey @eriknordmark @temerkhanov -- we need to do this (see my email on eve-tsc@) but we need to do this move properly. First of all we need to update our eve-alpine build cache and go from there. I plan to tackle it this week.

In the meantime @temerkhanov -- please let us know what exactly will break in your qemu work if we stay on 3.8 for now.

**Review by rvs — CHANGES_REQUESTED** [review]
Lets not merge this until we do a proper rebasing across the board in EVE

**Comment by temerkhanov** [comment]
> please let us know what exactly will break in your qemu work if we stay on 3.8 for now.

Everything should be more or less alright with 3.8

**Pull request merged**